Lombiq.Tests.UI/Extensions/ExtendedLoggingExtensions.cs

```diff
@@ -87,10 +92,41 @@ public static class ExtendedLoggingExtensions
         $"{operationName} applied to: {Environment.NewLine}{objectOfOperation}";
 
     private static void ExecuteSection(this UITestContext context, LogSection section, Action action) =>
-        context.Scope.AtataContext.Log.ExecuteSection(section, action);
+        context.Scope.AtataContext.Log.ExecuteSection(section, () =>
+        {
+            for (int i = 0; i < StabilityRetryCount; i++)
+            {
+                var notLast = i < StabilityRetryCount - 1;
+                try
+                {
+                    action();
+                    return;
+                }
+                catch (StaleElementReferenceException) when (notLast)
+                {
+                    Task.Delay(TimeSpan.FromSeconds(1)).Wait();
+                }
+            }
```

Comment:
Why not `throw new InvalidOperationException("Impossible to reach.");` here? Like below.

Reply:
As the text says, that's impossible to reach. The other methods need it because they expect to return a value so if I don't throw at the end then I get a compile time error even though it's really not a possible execution path.